prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -62,28 +65,25 @@ class ReportFunction : Logging {
             methods = [HttpMethod.POST],
             authLevel = AuthorizationLevel.FUNCTION
         ) request: HttpRequestMessage<String?>,
-        context: ExecutionContext,
+        context: ExecutionContext?
```

Comment:
out of scope: Could finally remove this? There's been a push to move over to the other logger

Reply:
Out of scope, but does need to be done - need to make WorkFlowEngine.kt inherit from Logging, and remove all references to context from it.